server_modules/sessions.js

```diff
@@ -0,0 +1,21 @@
+var args = require('./args');
+
+var session = require('express-session');
+var MemoryStore = session.MemoryStore;
+
+var config = {
+	store: new MemoryStore, 

```

Comment:
There is https://www.npmjs.com/package/session-file-store, which looks like a drop-in replacement and saves the session to disk. Can even be nice while developing, when using e.g. forever with watch mode (which will then constantly restart the server whenever a change is detected).